invenio_app_ils/ui/backoffice/src/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
```

Comment:
Not sure about the casing here, its the first time I see capitals on css, but it might be the "react" way. Even if its the react way, I am stating that because `css` are case sensitive, so I can only foresee a mess coming..

Reply:
This is automatically generated and it will be changed. Backoffice ui is just only what create-react-app generates so it will be changed :)